pytradfri/device.py

```diff
@@ -147,29 +147,6 @@ class LightControl:
     def __init__(self, device):
         self._device = device
 
-        self.can_set_mireds = None
-        self.can_set_color = None
-
-        if 'WS' in self._device.device_info.model_number:
-            self.can_set_mireds = True
-
-        if 'CWS' in self._device.device_info.model_number:
-            self.can_set_color = True
-
-        #  Define mireds range
-        if not self.can_set_mireds:
-            # White bulb
-            self._mireds_range = (None, None)
-        if not self.can_set_color:
-            # White spectrum bulb
-            self._mireds_range = RANGE_MIREDS_WS
-        if self.can_set_color:
-            # Color bulb
-            self._mireds_range = RANGE_MIREDS
-
-        self.min_mireds = self._mireds_range[0]
-        self.max_mireds = self._mireds_range[1]
-
```

Comment:
I've been quite rigorous here. I don't think it's the job of pytradfri to block sending values to lamps that ignore them.